config/locales/devise/fr.yml

```diff
@@ -122,7 +122,7 @@ fr:
       otp_delivery_preference:
         instruction: Vous pourrez changer votre choix la prochaine fois que vous vous
           connecterez
-        phone_unsupported: NOT TRANSLATED YET
+        phone_unsupported: NOT TRANSLATED YET %{location}
```

Comment:
What do you think about special-casing `NOT TRANSLATED YET` so we don't have to worry about adding these values in keys that get thrown out anyways?

Reply:
Hmm 🤔, I suppose we could do that.

Reply:
Yeah I think it would be nice -- because we keep the workflow of adding new keys simpler -- you can just `i18n-tasks add-missing -v "NOT TRANSLATED YET" --locales es,fr` and not worry about particular values

Reply:
Done. Mind taking one last look to let me know what you think of the change?